client/my-sites/plans/jetpack-plans/product-store/index.tsx

```diff
@@ -32,7 +36,11 @@ const ProductStore: React.FC< ProductStoreProps > = ( {
 				<IntroPricingBanner productSlugs={ productSlugs } siteId={ siteId ?? 'none' } />
 			</div>
 
-			<ViewFilter currentView={ currentView } setCurrentView={ setCurrentView } />
+			<ViewFilter
+				currentView={ currentView }
+				setCurrentView={ setCurrentView }
+				shouldUpdateUrl={ isValidViewQuery }
```

Comment:
I don't think this `shouldUpdateUrl` prop is needed and it can be removed because we want the `view` url query arg to always update _every_ time the user selects a view. 
Right now the way it works is the view url query only updates if the user lands on the pricing page with the view url query already present, but this is not ideal. The view url query should be added or updated _any_ time that either 'Products' or 'Bundles' is selected by the user.